app/src/db/migrations/20241002000000_011-ats-client-number.ts

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable max-len */
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return (
+    Promise.resolve()
+      // add ats ats_client_number to enquiry table
+      .then(() =>
+        knex.schema.alterTable('enquiry', (table) => {
+          table.text('ats_client_number');
```

Comment:
I can't quite remember, but is it nullable by default if not specified? If it is then ignore this comment.